tests/unit-core/test-api.c

```diff
@@ -1053,7 +1053,7 @@ main (void)
   {
     jerry_init (JERRY_INIT_EMPTY);
     jerry_value_t num_val = jerry_create_number (123);
-    jerry_value_set_error_flag (&num_val);
+    num_val = jerry_create_error_from_value (num_val, false);
```

Comment:
Are you sure this is false? You want to release `num_val` as well. Same above.

Reply:
You are right. I set it to true.